src/registry/storybook-helpers.tsx

```diff
@@ -10,9 +10,7 @@ export interface RenderArgs {
   onSubmit: FormioFormProps['onSubmit'];
 }
 
-type RenderFunc = Required<StoryObj<RenderArgs>>['render'];
-
-export const renderComponentInForm: RenderFunc = (args: RenderArgs, context) => (
+export const renderComponentInForm = (args: RenderArgs, context?: StoryContext<unknown>) => (
```

Comment:
Maybe better to use the `ArgsStoryFn<TRenderer, TArgs>` type from storybook. This makes the context type more explicit.

```suggestion
import type {ArgsStoryFn, Renderer} from 'storybook/internal/csf';
...
export const renderComponentInForm: ArgsStoryFn<Renderer, RenderArgs> = (args, context) => (
```